src/d3a/models/strategy/pv.py

```diff
@@ -188,3 +196,25 @@ def event_offer(self, *, market_id, offer):
         # if offer was deleted but not traded, free the energy in state.available_energy_kWh again
         if offer.seller == self.owner.name:
             self.state.available_energy_kWh[market.time_slot] -= offer.energy
+
+    def _set_alternative_pricing_scheme(self):
```

Comment:
Correct me if I am wrong, but do we really need this function? Isnt the rate dictated by the sellers in case of the alternative pricing? So IMO you can omit this function.

Reply:
EDIT: Ahhh, this is the PV. Of course we need it then. Sorry!